types/svg2png/tsconfig.json

```diff
@@ -2,12 +2,12 @@
     "compilerOptions": {
         "module": "commonjs",
         "lib": [
-            "es6",
-            "dom"
+            "es6"
         ],
         "noImplicitAny": true,
         "noImplicitThis": true,
         "strictNullChecks": false,
+        "allowSyntheticDefaultImports": true,
```

Comment:
I'd prefer to not enable this in tests and just use `import svg2png = require("svg2png")` instead.